docs/resources/new_in_doc.md

```diff
@@ -6,12 +6,81 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in Ibexa DXP documentation.
 
+## July 2023
+
+### v4.5.1
+
+- [v4.5.1 release notes](https://doc.ibexa.co/en/master/release_notes/ibexa_dxp_v4.5/#v451)
+
+### New home page
+
+- Redesigned [home page for the developer documentation](https://doc.ibexa.co/en/latest/)
+
+### Getting started
+
+- New cautions in [Install on Ibexa Cloud](https://doc.ibexa.co/en/master/getting_started/install_on_ibexa_cloud/) about using `cloud.ibexa.co` instead of and not `platform.sh`
```

Comment:
```suggestion
- New cautions in [Install on Ibexa Cloud](https://doc.ibexa.co/en/master/getting_started/install_on_ibexa_cloud/#2-create-an-account) about using `cloud.ibexa.co` instead of `platform.sh`
```